packages/service/src/service.ts

```diff
@@ -56,6 +56,19 @@ app.use(passport.session());
 
 app.post("/login", passport.authenticate("local", { successRedirect: "/" }));
 
+app.post("/users", async function (req, resp) {
+  const dbUser = await createUser(req.user);
+  const dbUserHashedPassword = await createPswash(req.user);
```

Comment:
Let's try to make this one database request. `upsert` user might not be the thing to do here anymore.

Create should just create, and error if the account already exists.